src/handlers/int-test-support/helpers/mock-data/csv/addFauxDataToTestPaths.ts

```diff
@@ -0,0 +1,25 @@
+export interface TestPath {
+  entity: string;
+  loa: string;
+  status: string;
+  path: "happy" | "sad";
+  eventId: string;
+  eventName: string;
+  clientId: string;
+}
```

Comment:
```suggestion
export interface TestPath {
  givenEntity: string;
  givenLoa: string;
  givenStatus: string;
  expectedPath: "happy" | "sad";
  expectedEventId: string;
  expectedEventName: string;
  expectedClientId: string;
}
```
makes it clearer what we do with these

Reply:
done